lib/tasks/reslugging.rake

```diff
@@ -99,7 +91,11 @@ namespace :reslug do
   - reindexes the org for search
   - reindexes all dependent documents in search"
   task :organisation, %i[old_slug new_slug] => :environment do |_task, args|
-    reslug_organisation(Organisation, args)
+    old_slug = args[:old_slug]
+    new_slug = args[:new_slug]
```

Comment:
I don't think these temporary variables are necessary - just pass the `args[:foo_slug]` directly where needed below.

Reply:
Sure, updated 👍

Reply:
Have you pushed? Still seeing the code here!

Reply:
Oops, there were more of them. Fixed now.